packages/flutter/lib/src/cupertino/theme.dart

```diff
@@ -0,0 +1,391 @@
+  /// Design where the [primaryColor] can appear on non-interactive surfaces like
+  /// the [AppBar] background, [TextField] borders etc.
+  ///
+  /// Reading this property from [CupertinoTheme.of] will create a dependency
+  /// from the [BuildContext] to changes in this property.
+  Color get primaryColor {
+    return _primaryColor ??
+        (_isLight ? CupertinoColors.activeBlue : CupertinoColors.activeOrange);
+  }
+
+  final Color _primaryContrastingColor;
+  /// A color used on content that contrast against a [primaryColor] background.
+  ///
+  /// This color is used on text and icons against a filled [CupertinoButton]
+  /// whose background is the [primaryColor] for instance.
+  ///
+  /// If coming from a Material [Theme] and unspecified, [primaryContrastingColor]
+  /// will be derived from the Material [ThemeData]'s `colorScheme.onPrimary`.
+  ///
+  /// Reading this property from [CupertinoTheme.of] will create a dependency
+  /// from the [BuildContext] to changes in this property.
+  Color get primaryContrastingColor {
+    return _primaryContrastingColor ??
+        (_isLight ? CupertinoColors.white : CupertinoColors.black);
+  }
+
+  final CupertinoTextTheme _textTheme;
+  /// Text styles used by Cupertino widgets.
+  ///
+  /// Derived from [brightness] and [primaryColor] if unspecified, including
+  /// [brightness] and [primaryColor] of a Material [ThemeData] if coming
+  /// from a Material [Theme].
+  ///
+  /// Reading this property from [CupertinoTheme.of] will create a dependency
+  /// from the [BuildContext] to changes in this property.
+  CupertinoTextTheme get textTheme {
+    return _textTheme ?? CupertinoTextTheme(
+      isLight: _isLight,
+      primaryColor: primaryColor,
+    );
+  }
+
+  final Color _barBackgroundColor;
+  /// Background color of the top nav bar and bottom tab bar.
+  ///
+  /// Defaults to a light gray or a dark gray translucid color depending
+  /// on the [brightness] when unspecified.
+  ///
+  /// Reading this property from [CupertinoTheme.of] will create a dependency
+  /// from the [BuildContext] to changes in this property.
+  Color get barBackgroundColor {
+    return _barBackgroundColor ??
+        (_isLight ? _kDefaultBarLightBackgroundColor : _kDefaultBarDarkBackgroundColor);
+  }
+
+  final Color _scaffoldBackgroundColor;
+  /// Background color of the scaffold.
+  ///
+  /// Defaults to white or black depending on the [brightness] if unspecified,
+  /// including [brightness] from a Material [ThemeData] if coming from a
+  /// Material [Theme].
+  ///
+  /// Reading this property from [CupertinoTheme.of] will create a dependency
+  /// from the [BuildContext] to changes in this property.
+  Color get scaffoldBackgroundColor {
+    return _scaffoldBackgroundColor ??
+        (_isLight ? CupertinoColors.white : CupertinoColors.black);
+  }
+
+  final Color _tableBackgroundColor;
+  /// Background color of a table view behind cell groups.
```

Comment:
Should this refer to a specific table component? What are "cell groups". Maybe just saying "the background color for tables" would be enough.

Reply:
It's a new widget Matt is making right now :) It's not in yet.